app/controllers/last_seen_issues_controller.rb

```diff
@@ -0,0 +1,31 @@
+class LastSeenIssuesController < ApplicationController
+
+    include QueriesHelper
+    helper :all
```

Comment:
I understand, but it will be beter to know exactly what helper we need. We don't need to add all the helpers (which can be a lot).

Can you search for the correct helper, and add it here?

Reply:
I can't. I tried to add just the `queries` helper which really contains `column_content()` what I've used, but probably without having `additional_tags` plugin active I wouldn't have thought about these cases where a new plugin patches the `column_content()` method and used another methods for display content.
![image](https://github.com/famiprog/redmine_last_seen_issues/assets/48293392/e31b66cc-1bdc-4ddf-8082-248b3cab07e5)
![image](https://github.com/famiprog/redmine_last_seen_issues/assets/48293392/08a5d1dd-47ba-4f16-acee-0dbeaed57350)
![image](https://github.com/famiprog/redmine_last_seen_issues/assets/48293392/20168191-7169-441b-905f-78287018dc42)

Reply:
Ok, I see.

Can you add a small comment on all ?

Reply:
Done.